lib/take_away.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/menu.rb'
+require './lib/message_sender.rb'
+
+class TakeAway
+  attr_reader :menu, :basket, :quantity, :dish
+
+  def initialize(menu = Menu.new.menu, message_sender = MessageSender.new)
+    @menu = menu
+    @basket = {}
+    @message_sender = message_sender
+  end
+
+  def read_menu
+    @menu
+  end
+
+  def order(dish, quantity = 1)
+    @dish = dish
+    @quantity = quantity
+    @basket[@dish] = @quantity
+    basket_message
+  end
+
+  def basket_message
+    "#{@quantity}x #{@dish}(s) added to your basket."
+  end
+  
+  def basket_summary
+    @basket.map do |dish, quantity|
+      if @menu.has_key?(dish)
+        "#{quantity}x #{dish}(s) = £#{@menu[dish] * quantity}"
```

Comment:
Love the addition of the quantity and the calculation of that

Reply:
When running the code in irb the total cost misses the last zero e,g £15.5 instead of £15.50. For user readability could you adjust the code to ensure this last digit always shows?